manifests/guestcluster/1.21/pvcsi.yaml

```diff
@@ -57,6 +57,9 @@ rules:
   - apiGroups: [""]
     resources: ["persistentvolumeclaims/status"]
     verbs: ["update", "patch"]
+  - apiGroups: [ "cns.vmware.com" ]
```

Comment:
no need to update pvcsi manifest before 1.22 as TKGS HA will only be supported in TKR 1.22.